frontend/src/app/layouts/projects/project/project.component.ts

```diff
@@ -33,7 +35,9 @@ export class ProjectComponent implements OnInit {
     }
 
     run() {
-        this.result = `data:text/html;charset = utf - 8,<html>
+        this.result = `${this.frameType}`;
+        setTimeout(() => {
```

Comment:
Why do we need `setTimeout` here?

Reply:
This requires that the code is inside, falling to the end of the execution stack

Reply:
What the problem you're facing? Why this doesn't work without timeout?

Reply:
the problem is when you click at run button second time without changing the fields, our js code does not run  again. It is happening that I use pipe, so to fix this problem use setTimeout.